core/templates/services/site-analytics.service.ts

```diff
@@ -71,9 +71,14 @@ export class SiteAnalyticsService {
       'GuideParentsButton', 'click',
       this.windowRef.nativeWindow.location.pathname);
   }
-  registerClickTipforParentsButtonEvent(): void {
+  registerClickStartContributingButtonEvent(): void {
```

Comment:
Why this change? We still need the `registerClickTipforParentsButtonEvent` function, right? any specific reason for moving this and other functions in this file above/below?

[In general, try to have the least changes in the files and avoid touching the lines which re not expected to change. That will help us to keep the history of the file/lines intact and easy to check.]

Reply:
@ReshuKumari Is this comment resolved?

Reply:
yes resolved